modules/custom/az_alphabetical_listing/az_alphabetical_listing.module

```diff
@@ -41,3 +104,86 @@ function az_alphabetical_listing_theme_registry_alter(&$theme_registry) {
     }
   }
 }
+
+/**
+ * Implements hook_views_post_execute().
+ */
+function az_alphabetical_listing_views_post_execute(ViewExecutable $view) {
+  $entity_type_id = $view->getBaseEntityType()->id();
```

Comment:
This seems like it should be moved inside the if statement so that it does not run for every view.